pkg/storage/replicate_queue.go

```diff
@@ -94,6 +98,7 @@ func makeReplicateQueueMetrics() ReplicateQueueMetrics {
 // additional replica to their range.
 type replicateQueue struct {
 	*baseQueue
+	options           replicateQueueOptions
```

Comment:
I don't think we'll ever want this field to be set, so I'd just remove it.

Reply:
Done.